Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Update the base image in Makefile when updating golang version. This has to
 # be pre-pulled in order to work on GCB.
 ARG ARCH
-FROM golang:1.17.13 as build
+FROM golang:1.19.5 as build
```

Comment:
the release of 0.6 will only release a patch version in the future. Not suitable for major upgrade changes
/cc @serathius @dgrisonnet

Reply:
Closing this PR therefore. Opened another PR https://github.com/kubernetes-sigs/metrics-server/pull/1194 where we stick to golang:1.17 although 1.17 is not getting new security fixes.